src/models/streaming/AudioConfig.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Configuration for an audio stream.
+ */
+export class AudioConfig {
+    contentType: string;
+    layout?: string;
+    rate?: number;
+    format?: string;
+    channels?: number;
+
+    /**
+     * @param contentType {string} Content type of the audio stream.
+     * @param layout (optional) {string} Layout of the audio channels. "interleaved" or "non-interleaved".
+     * @param rate (optional) {int} Sample rate of audio. 8000-48000hz supported.
+     * @param format (optional) {string} Format of audio.
+     * @param channels (optional) {int} Number of audio channels. 1-10 supported.
```

Comment:
Does google document type?

I find it strange that we document types for two reasons:
1. int type does not explicitly exist in JS, but if Google is doing this in their docs, I'm fine with doing it in ours.
2. We already have type definitions as part of typescript. Do documentation comments end up in the transpiled js files found in the distribution?

Reply:
1. You're right, I should have been using number. 
2. That's fair, we do include the .d.ts files in the distribution. Google does not write in typescript so that's most likely why they include types. I will remove them